joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/utils/PCodeMapper.scala

```diff
@@ -449,7 +449,7 @@ class PCodeMapper(
     }
   }
 
-  def connectCallToArgument(call: CfgNodeNew, argument: CfgNodeNew): Unit = {
+  def _connectCallToArgument(call: CfgNodeNew, argument: CfgNodeNew): Unit = {
```

Comment:
`_` prefix is usually used to private internal hidden stuff, but this is a public method - why did you make that change?